bug_report_page.php

```diff
@@ -605,7 +605,22 @@
 			<label for="attach_tag"><?php echo lang_get( 'tag_attach_long' ) ?></label>
 		</th>
 		<td>
-			<?php print_tag_input( '' ); ?>
+			<?php
+				if( $f_master_bug_id > 0 ) {
+					# pre-fill tag string when cloning from master bug
+					$t_tags = [];
+					foreach( tag_bug_get_attached( $f_master_bug_id ) as $t_tag ) {
+						array_push( $t_tags, $t_tag["name"] );
+					}
+					$t_tag_string = implode(
+						config_get( 'tag_separator' ), $t_tags
+					);
+					print_tag_input( $f_master_bug_id, $t_tag_string );
+				} else {
+					# otherwise show just the default empty string
+					print_tag_input( '' );
```

Comment:
Didn't test anything, but seems this call was / is wrong as the first parameter is the bug id
```
/**
 * Print the separator comment, input box, and existing tag dropdown menu.
 * @param integer $p_bug_id A bug identifier.
 * @param string  $p_string Default contents of the input box.
 * @return void
 */
function print_tag_input( $p_bug_id = 0, $p_string = '' )
```

Reply:
Actually this API call has been like this since it its introduction back in 2015 (see f67ec45972ada6ef3cf3b727b41ce1afb967ff6c).

This has no impact on functionality, as the string is converted to (bug_id) 0, meaning that all available tags are included in the selection list.